Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "rust-g"
 edition = "2021"
-version = "2.1.0"
+version = "3.0.0"
```

Comment:
normally done for the release tag PR version (ex. https://github.com/tgstation/rust-g/pull/143), but can do here.